src/sql/parts/notebook/notebookInput.ts

```diff
@@ -154,6 +156,10 @@ export class NotebookInput extends EditorInput {
 		this.assignProviders();
 	}
 
+	public get notebookEditorInput(): UntitledEditorInput {
```

Comment:
This name is a bit confusing - can we consider something like `textInput` as this overlaps with NotebookInput and NotebookTextModel as-is?